pyproject.toml

```diff
@@ -1,3 +1,7 @@
+[build-system]
+requires = ["poetry-core"]
+build-backend = "poetry.core.masonry.api"
+
 [tool.poetry]
 name = "llama-parse"
 version = "0.4.0"
```

Comment:
should we bump up the version?

Reply:
Going to do that im a separate PR, just want to test the publishing flow on its own